lmdeploy/model.py

```diff
@@ -53,7 +47,13 @@ def chat_template(self):
             for key, value in dataclasses.asdict(self).items()
             if value is not None
         }
-        model: BaseModel = MODELS.get(self.model_name).from_config(**attrs)
+        if self.model_name in MODELS.module_dict.keys():
+            model: BaseModel = MODELS.get(self.model_name)(**attrs)
+        else:
+            logger.warning(
+                f'Could not find {self.model_name} in registered models. '
```

Comment:
f"unable to locate '{self.model_name}' in registered chat templates, falling back to the default template. Please register a new chat template if the default one does not meet your requirements”

Reply:
This is for json template usage. Any new name will be treated as newly registered.